README.md

```diff
@@ -8,15 +10,19 @@ It currently does not yet implement many important features, such as SOCKS,
 HTTP/2, and many of the predefined JS functions that PAC files use.
 Contributions are welcome, please reach out to me if you'd like to help!
 
-## Installation and Usage
+## Download
+
+Alpaca can be downloaded from the [github releases page][1].
 
-To download and install Alpaca, use:
+## Install From Source
```

Comment:
I'm in two minds about this. I like to know that a go program can be installed with `go get` - too many cant - they need a makefile or some generate step. Also, people are looking at the README from the source code - i think the programmer is the primary audience of that. Other documentation can be more user-oriented.

However I think it would help make it less ambiguous if the "Download" section was called "Install from GitHub Release" or something, so the "Install from Source" stands out as an alternate installation option. Having a Download then Install section will lead the less discerning user to try one after the other I would expect.

Perhaps the "Install from GitHub Release" should also say a bit more than download the release. Unpack and put on your path is also needed, in whatever detail that would be required for the audience.

Reply:
OK, what if we called the sections "Download Binary" and "Build from Sources"? Hopefully that should be obvious enough.

I agree that it's not always obvious that a Go program is `go get`-able, but given the way GitHub displays the README file on the project page, this is the closest thing to a "home page" that Alpaca has, which is why I think of this as a user-oriented file.